sdk/core/src/test/java/com/klaviyo/core/RegistryTest.kt

```diff
@@ -10,6 +14,17 @@ class RegistryTest {
     private interface TesWrongDependency
```

Comment:
Just noticed this very minor should it `Test` instead of `Tes`?

Reply:
Ya, I think I caught it on one of the other PRs